package.json

```diff
@@ -10,16 +10,15 @@
     "http": "stream-http"
   },
   "scripts": {
-    "test": "aegir test",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser webworker",
+    "test": "aegir test --no-parallel --timeout 80000",
```

Comment:
Which tests still require increased timeout?

Reply:
not sure..ci was just being slow and I didn't want to spend an hour on going test by test on ci